src/features/licenses/components/LicenseForm/index.jsx

```diff
@@ -66,6 +66,10 @@ export const LicenseForm = ({
   const [optionSelection, setOptionSelection] = useState(created ? selectorOptions.courses : '');
 
   const showCatalogSelector = getConfig().SHOW_CATALOG_SELECTOR || false;
+  const isMultiCatalogSel = getConfig().MULTI_CATALOG_SELECTOR || false;
+
+  const disableLicenseNameIn = (created && !isMultiCatalogSel)
+  && (selectedCatalogs.length > 0 || optionSelection === selectorOptions.catalogs);
```

Comment:
This could be more easy to read if:

```javascript
// Is just an example, you can a better names for these variables
const hasCatalogSelected = selectedCatalogs.length > 0 || optionSelection === selectorOptions.catalogs;
const disableLicenseNameInput = created && !isMultiCatalogSel && hasCatalogSelected;

```